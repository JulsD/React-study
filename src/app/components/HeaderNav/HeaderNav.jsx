import React from 'react';
import Logo from './../Logo/Logo';
import NavMenu from './../NavMenu';

import styles from './header-nav.css';

class Nav extends React.Component {

  constructor(props) {
    super(props);
  }

  render() {
    const movieOpend = this.props.movieOpend;

    let navBody = null;
    if (movieOpend) {
      navBody = <NavMenu menu={['Search']}/>;
    }
    
    return (
      <nav className = {styles.root}>
        <Logo />
        {navBody}
      </nav>
    );
  }

}

export default Nav;
