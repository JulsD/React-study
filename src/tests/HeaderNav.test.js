import React from 'react';
import HeaderNav from '../app/components/HeaderNav';

test('HeaderNav should render a <nav>', () => {
  const wrapper = shallow(
    <HeaderNav />
  );
  expect(wrapper).toMatchSnapshot();
});
