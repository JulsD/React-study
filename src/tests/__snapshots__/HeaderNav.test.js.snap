// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderNav should render a <nav> 1`] = `
<nav
  className="root"
>
  <Logo />
</nav>
`;
